content/en/threat-model.md

```diff
@@ -0,0 +1,123 @@
+|                        | x                |               |                       | x            |               |              |             | Fulcio CA server + Rekor compromise capabilities. The recovery is harder than just Fulcio CA server or Rekor server compromise because the history of both tables may be lost.                                                                                                                                                                                                                                                                                                                                                                           |
+|                        | x                | x             | x                     |              |               |              |             | Same as [Fulcio CA server + Fulcio CT], but replay and fork attacks will not be reported by the monitors.                                                                                                                                                                                                                                                                                                                                                                                                                                                |
+|                        |                  |               |                       |              |               | x            |             | Arbitrary software attack for whatever that specific account has access to, might be recoverable through a mechanism set up by the OIDC server.                                                                                                                                                                                                                                                                                                                                                                                                          |
+|                        |                  | x             |                       |              |               | x            |             | Same as Fulcio CT log + OIDC account                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
```

Comment:
Again, that's simply the set of things compromised on this row.  Is this trying to say, "each of the two above vulnerabilities"?

With 8 components, I'm assuming we don't want a 256-row matrix (or 129 if we consider all the root-of-trust breaches equally catastrophic).  I'd suggest romeving the "Same as X" rows unless there's additional attacker capabilities enabled by the interaction between components.